docs/general-knowledge/flight_rules.md

```diff
@@ -0,0 +1,48 @@
+# Flight Rules
+
+Flight rules are the rules, regulations & procedures used in aircraft flying in various conditions. The most common include:
+
+- VFR (Visual Flight Rules)
+- IFR (Instrument Flight Rules)
+
+## VFR
+
+VFR stands for 'Visual Flight Rules', and as the name infers, this means flying using visual references. This can be a very simple navigation method of navigation, though requires Visual Meteorological Conditions (VMC). The VMC visibility and distance from cloud minima are contained in the table below (Table 3-1, [ICAO Annex 2](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiY2pqIl9T-AhXLTcAKHXhmDEAQFnoECBgQAQ&url=https%3A%2F%2Fwww.icao.int%2FMeetings%2Fanconf12%2FDocument%2520Archive%2Fan02_cons%255B1%255D.pdf&usg=AOvVaw3TZDU3bcTXjQcuw_2jfQ0y))
+
+
+![VMC Minima Chart](../assets/VMC_Minima.png)
+
+Weather conditions can be found in METARs (See [meteorology](./meteorology.md) for more details.)
+
+If these minima are not met, then the conditions are considered Instrument Meterological Conditions (IMC), and flights are required to opperate under IFR. VFR also requires the pilot to see and avoid obsticles, such as terrain or other aircraft and pilots are responsible for seperation with other aircraft. Though seperation is down to the pilot, Air Traffic Control may require the use of a transponder to identify the aircraft and provide seperation from IFR aircraft (See below). If an aircraft is planning to arrive at an airport in IMC using VFR, this aircraft will be denied clearance, and will have to divert or declare an emergency.
+
+!!! note Note VFR flights cannot opperate in 'Class A' airspace, as this requires the pilot to be flying IFR, though may be entered if an emergency is decleared, or if a pilot is avoiding severe weather formations (This should be avoided)
+
+!!! note Note VFR is the flight rule **always** used by helicopters, though with some alterations, given due to the unique operating characteristics of rotary-wing aircraft.
+
+## IFR
+
+IFR stands for 'Instrument Flight Rules', and this means solely refering to the instruments for navigation. This is more complicated than VFR, and flights can take place in any weather conditions, so limitations are mainly down to the aircraft's limits. As IFR flights can take place in any weather, seperation is down to Air Traffic Control, as pilots no longer can see obsticles.
+
+As IFR flights do not use Visual References, IFR flights use navigational aids, such as VORs, NDBs, as well as GPS/GNSS. Air Traffic Control can also assign headings/radar vectors for the aircraft to fly.
+
+!!! example ATC BAW134 Fly heading Zero Seven Zero.

```

Comment:
Change to BAW134 Tower Fly heading Zero Seven Zero

Radio calls follow the structure of "Hey you, it's me, *intentions*"